src/UI/components/upload/validation/validationItem.test.tsx

```diff
@@ -0,0 +1,60 @@
+import { render } from '../../../test_config/render';
+import { fireEvent, screen } from '@testing-library/dom';
+import '@testing-library/jest-dom';
+import ValidationConsole from './validationConsole';
+import { AppState } from '../../../state/store';
+import { initialState } from '../../../state/upload/uploadSlice';
+import { useAppSelector } from '../../../state/hooks';
+
+jest.mock('../../../state/hooks', () => ({
```

Comment:
You shouldn't have to mock the useAppSelector - just pass in the state in the render method with what you want to come out of it.

Reply:
It's not being used in this component anyway... Why is it here?